app/models/user.rb

```diff
@@ -121,6 +123,12 @@ def assigned_to_postcode?(postcode)
     owned_neighbourhood_ids.include?(neighbourhood&.id)
   end
 
+  def partnership_tag?
+    return true if tags.all? { |tag| tag.type == 'Partnership' }
+
+    errors.add(:tags, 'Can only be of type Partnership')
```

Comment:
This error is a bit arbitrary because as you say, they should never be able to do this. this PR limits the ability to attempt this in the UI, but if someone accidentally changes it so they have access to more tags again hopefully it will explain what the problem is?

Is there a suggestion for how the error could be more useful in the unfortunate event someone encounters it?

Reply:
not really- it is good that we are validating this (for us) but yeah- it shouldn't be possible to do this from the UI in the first place